mod/groups/actions/groups/edit.php

```diff
@@ -210,21 +210,7 @@
 	));
 }
 
-$has_uploaded_icon = (!empty($_FILES['icon']['type']) && substr_count($_FILES['icon']['type'], 'image/'));
-
-if ($has_uploaded_icon) {
-	$filehandler = new ElggFile();
-	$filehandler->owner_guid = $group->owner_guid;
```

Comment:
i believe this breaks already uploaded images... that is why https://github.com/Elgg/Elgg/issues/4683 exists... fine by me to merge this now... upgrade can fix the old locations.